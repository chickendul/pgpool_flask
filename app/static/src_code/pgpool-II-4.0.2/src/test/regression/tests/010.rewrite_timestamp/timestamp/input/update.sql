version https://git-lfs.github.com/spec/v1
oid sha256:2c6462c2b025afb06694ec2bb1775ef181f933f269f5e16434a6226ea0e7e831
size 306
