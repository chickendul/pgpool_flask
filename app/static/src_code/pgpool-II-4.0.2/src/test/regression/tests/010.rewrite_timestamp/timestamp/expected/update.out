version https://git-lfs.github.com/spec/v1
oid sha256:148fcb4dbe0eb818e3aa485505eecd8e13609c4b62eee405fbbf5066cfa5c6bb
size 602
